projects/storefrontapp-e2e-cypress/cypress/integration/regression/site-context/language/language-cart-page.e2e-spec.ts

```diff
@@ -1,4 +1,4 @@
-import * as cart from '../../../../helpers/cart';
+import { manipulateCartQuantity } from '../../../../helpers/cart';
 import * as siteContextSelector from '../../../../helpers/site-context-selector';
 
 describe('Language switch - cart page', () => {
```

Comment:
Don't require author to fix code that wasn't changed or isn't related to PR fix/feature. Better create a ticket for it.

Reply:
General rule from Google guidelines:

> In general, reviewers should favor approving a CL once it is in a state where it definitely improves the overall code health of the system being worked on, even if the CL isn’t perfect.